crates/common_utils/Cargo.toml

```
@@ -48,6 +47,9 @@ md5 = "0.7.0"
 masking = { version = "0.1.0", path = "../masking" }
 router_env = { version = "0.1.0", path = "../router_env", features = ["log_extra_implicit_fields", "log_custom_entries_to_extra"], optional = true }
 
+[target.'cfg(not(target_os = "windows"))'.dependencies]
+signal-hook-tokio = { version = "0.3.1", features = ["futures-v0_3"], optional = true }
```

```suggestion
[target.'cfg(not(target_os = "windows"))'.dependencies]
signal-hook-tokio = { version = "0.3.1", features = ["futures-v0_3"] }
```

I believe this would have to be changed as well, in case CI fails again.

---

Nevermind, the `signals` module in `common_utils` is behind a feature flag on its own. Resolving this conversation.